Mathlib/Algebra/DirectSum/Internal.lean

```diff
@@ -276,6 +276,56 @@ theorem coe_of_mul_apply {i : ι} (r : A i) (r' : ⨁ i, A i) (n : ι) [Decidabl
 
 end CanonicallyOrderedAddCommMonoid
 
+section GradeZeroSubsemiring
+
+variable [Semiring R]
+variable [CanonicallyOrderedAddCommMonoid ι]
```

Comment:
You're not actually using this order, right? I think `AddMonoid` would be sufficient here.

Reply:
Yeah, `AddMonoid` is enough!